src/test/java/com/keyboardbuilder/enterprise/dto/KeySwitchesTest.java

```diff
@@ -0,0 +1,55 @@
+package com.keyboardbuilder.enterprise.dto;
```

Comment:
2. Simple JUnit Test for KeySwitches. Given the use of Lombok functions in the DTO classes, the function names should be correct.

Reply:
Nice.